chart/templates/_helpers.tpl

```diff
@@ -258,7 +258,7 @@ env:
 {{- $gp := .gp -}}
 {{- $comp := .comp -}}
 {{- $tracing := $comp.tracing | default $gp.tracing -}}
-{{- if $tracing }}
```

Comment:
I think this might be a breaking change. I believe that all users of the Helm chart that previous has tracing enabled by setting `tracing.endpoint` would now have to update their values files to also have `tracing.enabled`

If so I would rather no go down this route and suggested in https://github.com/gitpod-io/ops/issues/334 and just use additional values files for preview environments.

Reply:
I change it, but it is common to use `.enabled` to switch optional deployments.

Reply:
Done